README.md

```diff
@@ -95,10 +47,9 @@ Create the log directory and log file
 [~/nipovpn]>$ sudo touch /var/log/nipovpn/nipovpn.log 
 ```
 
-#### Run
 Run it
 ```bash
-[~/nipovpn]>$ sudo nipovpn/usr/bin/nipovpn server nipovpn/etc/nipovpn/config.yaml
+sudo nipovpn/usr/bin/nipovpn server nipovpn/etc/nipovpn/config.yaml
```

Comment:
This line should be as following, since the binary file is in build/core/
```
sudo build/core/nipovpn_core server nipovpn/etc/nipovpn/config.yaml
```

Reply:
Done

Reply:
Thanks